### PR TITLE
apps/userdashboard/moderationNotification/ActionsBar/HoverButton: ref…

### PR DESCRIPTION
**Pull request by khamui**
…actor to show hoverbutton instantly (without loading).

To mention: the disabling is still a little bit delayed because this is set by a variable which has to come from the `list re-fetching`.

### CODE REVIEW

**Comment by khamui** [comment]
In order to test "hover" through tabbing, you might want to set the style `outline` on focus by hand (this comes from a rebase of aplus).

**Comment by khamui** [comment]
@phillimorland @goapunk after thinking again about this i decided to remove a huge complexity (this criteria stuff), which i shifted into the HoverButton component before. But it is definitely not the responsiblity of the button to deal with any criterias.

So instantly changing the button state is not possible by design, because we fetch this information nested in a parent list. I think twitter fetches the `follow` state immediately (inside of the component) on an endpoint called `/friendships`, but not within a parent list. That is why they can set it instantly. For us this would be a hack (with this complexity we had before).

long story short: I think in order to have clean code we have to deal with the time the list gets fetched again. Currently i am `disabling` the button while fetching.

**Review by phillimorland — APPROVED** [review]
yo yo much cleaner code, nice one, the functionality might need a little tweeking but I think thats a choice for Marcelo and we won't know the actual load speed unless we try in on a live site so think we should merge and see :)
